hdfs-connector/src/main/java/com/oracle/bmc/hdfs/BmcFilesystem.java

```diff
@@ -895,10 +909,6 @@ public synchronized void close() throws IOException {
             isClosed = true;
         } catch (Exception e) {
             LOG.warn("Caught exception while closing filesystem", e);
-        } finally {
```

Comment:
Q : 1. Why Close the holders when underlying Impl which is cache is closed ?
 2. Shouldn't the holding object release the resource when they are done with it ?